pycsw/wsgi.py

```diff
@@ -1,28 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
The removal of the copyright and license header from `pycsw.wsgi` is not acceptable. Please re-add this information to the file. You may add yourself as an author, pending the acceptance of this PR.